lib/utilities/lang.js

```diff
@@ -2,7 +2,9 @@
 // Thanks Yahoo!
 // http://yuilibrary.com/yui/docs/api/files/yui_js_yui-lang.js.html
 
-var TOSTRING     = Object.prototype.toString;
+/*eslint-disable */
```

Comment:
I think this file should be modified to be in line with our style. It's not completely a direct copy-paste. I thought I already fixed several things.